examples/apt/bullseye.yaml

```diff
@@ -0,0 +1,33 @@
+# Packages for examples/apt.
+#
+#  Anytime this file is changed, the lockfile needs to be regenerated.
+#
+#  To generate the bullseye.lock.json run the following command
+#
+#     bazel run @bullseye//:lock
+#
+# See debian_package_index at WORKSPACE.bazel
+version: 1
+
+sources:
+  - channel: bullseye main
+    url: https://snapshot-cloudflare.debian.org/archive/debian/20240210T223313Z
```

Comment:
is it possible to have multiple URLs - one that's more reliable and a second that's more canonical/guaranteed?

Reply:
can could be a future implementation, we'll be iterating over this a lot before 1.0. i'd rather leave that as a follow-up/future work.